__tests__/app.test.js

```diff
@@ -12,6 +12,17 @@ afterAll(() => {
   return db.end();
 });
 
+describe("GET /api", () => {
+  it("should return the endpoints object", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        expect(response.statusCode).toBe(200);
+        expect(response.body).toEqual(expect.any(Object));
```

Comment:
We don't want this to be any object - we want to make sure it's the object we're looking for. You have two options here:
- bring in the endpoints file and test it's equal to your response
- iterate over the response and check the keys are right i.e. slug and description